staging/src/k8s.io/api/autoscaling/v2beta1/types.go

```diff
@@ -294,6 +320,14 @@ type PodsMetricStatus struct {
 	// currentAverageValue is the current value of the average of the
 	// metric across all relevant pods (as a quantity)
 	CurrentAverageValue resource.Quantity `json:"currentAverageValue" protobuf:"bytes,2,name=currentAverageValue"`
+
+	// selector is the selector for the given metric
+	// it is the string-encoded form of a standard kubernetes label selector
+	// +optional
+	Selector *metav1.LabelSelector `json:"selector,omitempty" protobuf:"bytes,3,name=selector"`
+	// value is the current value of the metric (as a quantity)
+	// +optional
+	Value *resource.Quantity `json:"value,omitempty" protobuf:"bytes,4,name=value"`
```

Comment:
You need to explain how this is different from CurrentAverageValue.  Reading the godoc, I have no idea.

Reply:
This was actually added in error -- `averageValue` is sufficient for pods metrics. Thanks for catching, removed